addons/rangecard/functions/fnc_updateRangeCard.sqf

```diff
@@ -145,38 +151,38 @@ if (missionNamespace getVariable [QEGVAR(advanced_ballistics,enabled), false]) t
 
 ctrlSetText [77003, format["%1m ZERO", round(_zeroRange)]];
 
-if (missionNamespace getVariable [QEGVAR(advanced_ballistics,enabled), false]) then {
+if (_isABenabled) then {
     ctrlSetText [770001, format["Drop Tables for B.P.: %1mb; Corrected for MVV at Air/Ammo Temperatures -15-35 °C", round(EGVAR(scopes,zeroReferenceBarometricPressure) * 100) / 100]];
     ctrlSetText [77004 , format["B.P.: %1mb", round(EGVAR(scopes,zeroReferenceBarometricPressure) * 100) / 100]];
 } else {
     ctrlSetText [770001, ""];
     ctrlSetText [77004 , ""];
 };
 
-private _cacheEntry = missionNamespace getVariable format[QGVAR(%1_%2_%3_%4_%5), _zeroRange, _boreHeight, _ammoClass, _weaponClass, missionNamespace getVariable [QEGVAR(advanced_ballistics,enabled), false]];
-if (isNil {_cacheEntry}) then {
-    private _scopeBaseAngle = if (!_useABConfig) then {
+private _cacheEntry = missionNamespace getVariable format [QGVAR(%1_%2_%3_%4_%5_%6_%7), _zeroRange, _boreHeight, _ammoClass, _weaponClass, _isABenabled, _useBarrelLengthInfluence, _useAmmoTemperatureInfluence];
+if (isNil "_cacheEntry") then {
+    private _scopeBaseAngle = if (!_isABenabled) then {
```

Comment:
```suggestion
    private _scopeBaseAngle = if ((!_isABenabled) || {_bc == 0}) then {
```
I think this might be needed to match original behavior? (old line 112)

Reply:
updated in _isABenabled bool instead